schema/rdl_schema_0.1.json

```
@@ -1710,13 +1552,502 @@
+          ],
+          "codelist": "function_approach.csv",
+          "openCodelist": false
+        },
+        "relationship": {
+          "title": "Impact relationship type",
+          "type": "string",
+          "description": "The type of function relationships used to calculate the impact values, taken from the closed [relationship type codelist](https://rdl-standard.readthedocs.io/en/{{version}}/data_model/codelists/#relationship_type).",
+          "enum": [
+            "discrete values",
+            "mathematical function (bespoke)",
+            "mathematical function (parametric)"
+          ],
+          "codelist": "relationship_type.csv",
+          "openCodelist": false
+        }
+      },
+      "minProperties": 1
+    },
+    "Fragility_function": {
+      "title": "Fragility function",
+      "type": "object",
+      "description": "Details of a fragility function used to calculate the impact of the hazard.",
+      "properties": {
+        "approach": {
+          "title": "Impact function approach",
+          "type": "string",
+          "description": "The approach the impact function is based upon, taken from the closed [function approach codelist](https://rdl-standard.readthedocs.io/en/{{version}}/data_model/codelists/#function_approach).",
+          "enum": [
+            "analytical",
+            "empirical",
+            "hybrid",
+            "judgement"
+          ],
+          "codelist": "function_approach.csv",
+          "openCodelist": false
+        },
+        "relationship": {
+          "title": "Impact relationship type",
+          "type": "string",
+          "description": "The type of function relationships used to calculate the impact values, taken from the closed [relationship type codelist](https://rdl-standard.readthedocs.io/en/{{version}}/data_model/codelists/#relationship_type).",
+          "enum": [
+            "discrete values",
+            "mathematical function (bespoke)",
+            "mathematical function (parametric)"
+          ],
+          "codelist": "relationship_type.csv",
+          "openCodelist": false
+        },
+        "damage_scale_name": {
+          "title": "Damage scale name",
+          "type": "string",
+          "description": "The name of the damage scale used in the damage-to-loss function or fragility function, taken from the open [damage scale name codelist](https://rdl-standard.readthedocs.io/en/{{version}}/data_model/codelists/#damage_scale_name).",
+          "codelist": "damage_scale_name.csv",
+          "openCodelist": true,
+          "minLength": 1
+        },
+        "damage_states_names": {
+          "title": "Damage states names",
+          "type": "array",
+          "description": "The names of the damage states listed in a Damage Scale or used in a Fragility function, taken from the open [damage states names codelist](https://rdl-standard.readthedocs.io/en/{{version}}/data_model/codelists/#damage_states_names).",
```

Update codelist name for consistency with other codelists, which aren't plural.

```suggestion
          "description": "The names of the damage states listed in a damage scale or used in a fragility function, taken from the open [damage state name codelist](https://rdl-standard.readthedocs.io/en/{{version}}/data_model/codelists/#damage_state_name).",
```

---

see previous reply

---

although agree that names should be singular in the codelist so I've updated that

---

Resolving since the codelist was removed in a later commit.